paper/paper.md

```diff
@@ -352,6 +354,11 @@ resource_manager:
     namespace: mynamespace
 ```
 
+By setting up CI for Popper workflows, users can continuously validate changes made to their workflows and also protect their workflows from getting outdated due to reasons such as outdated dependencies, outdated container images, and broken links.
+
+A summary of the training duration and accuracy obtained by running the workflow in three different computing environment is shown in @Fig:casestudies.
```

Comment:
we can have a "Results" section . We could move the last paragraphs of each previous subsection (where we talk about performance gains of each approach) to here.